manifests/squid.yaml.in

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: spice-proxy
+spec:
+  ports:
+    - port: 3128
+      targetPort: spice-proxy
+  externalIPs:
+    - "{{ master_ip }}"
```

Comment:
For a new patch: We can really start pulling out master_ip from all yamls and populate it from ENV vars, then we can use a ConfigMap to define it (we effectively then just need ot create a configmap for the cluster).